### DISCUSSION TITLE
Is a EMF diagram editor example available?

### DISCUSSION THREAD

**Discussion by Deel96** [eclipse-emfcloud/emfcloud#162] [Coffee Editor] [answered]
Hi,
on the homepage of EMF.cloud there is EMF editing displayed.
The coffee editor has a flow chart editor, but not a EMF editor.
Is there also a example available with a emf diagram editor?

Best Regards

**Comment by eneufeld** [comment, marked as answer]
Hi @Deel96 ,
yes you can find it here: https://github.com/eclipse-emfcloud/ecore-glsp

**Reply by Deel96** [reply]
Thank you, somehow I missed that. I tried to run it, but I get errors when moving diagram elements. I made a issue about it at the repository: https://github.com/eclipse-emfcloud/ecore-glsp/issues/125
Or should I made a post in the discussion panel?

**Reply by eneufeld** [reply]
Hi @Deel96 ,
the issue is fine, we will take a look.
